docs/wiki/_Sidebar.md

```diff
@@ -18,7 +18,7 @@
     - [Basic][wiki_starter_module_basic]
     - [Hub Networking][wiki_starter_module_hubnetworking]
     - [Complete][wiki_starter_module_complete]
-      - [YAML Schema for `config.yaml`][wiki_yaml_schema_reference]
+      - [YAML Schema for configuration file][wiki_yaml_schema_reference]
```

Comment:
This will probably fold onto a second line as we have limited space in the side bar.
```suggestion
      - [Config file YAML schema][wiki_yaml_schema_reference]
```

Reply:
opted for configuration to match the variable name for consistency `configuration_file_path`